schema/admin-stats-response.json

```diff
@@ -115,6 +115,14 @@
                 "checksum": {
                     "type": "number"
                 },
+                "checksums": {
```

Comment:
Since ringpop-node doesn't support it yet, `checksums` is not added to the list with required properties.